tls/crypto.c

```diff
@@ -161,29 +161,29 @@ typedef struct {
 static TlsMdInfo ttls_sha224_info = {
 	.type		= TTLS_MD_SHA224,
 	.name		= "SHA224",
-	.alg_name	= "sha224-avx2",
-	.hmac_name	= "hmac(sha224-avx2)",
+	.alg_name	= "sha224",
```

Comment:
@i-rinat has explained in chat about priorities in Linux crypto, why this solution works and why it's best option. Please add the comment before `ttls_sha224_info`